docs/cmdline-opts/ssl-disable-tls13-middlebox.d

```diff
@@ -0,0 +1,5 @@
+Long: ssl-disable-tls13-middlebox
+Help: Disable middlebox option for tls 1.3 
+Added: 7.25.0
+---
+This option tells curl to not use middlebox for tls 1.3.
```

Comment:
This should mention that it only applies to the OpenSSL backend.

Reply:
Yes I agree.